src/main/scala/cromwell/engine/backend/jes/JesAttributes.scala

```diff
@@ -31,6 +32,7 @@ object JesAttributes {
     "applicationName",
     "project",
     "baseExecutionBucket",
+    "scratchBucket",
```

Comment:
Hmm baseExecutionBucket is where everything happens (call directories, logs, outputs, exec.sh...). What this PR does is use user's crendentials (if available) for all interactions with this bucket.
The scratchBucket still uses Cromwell credentials (service account) and is only used to upload/delete the auth file and users shouldn't have access to it.
So both are needed I think.